Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSourceSmall.java

```diff
@@ -0,0 +1,138 @@
+package org.mage.plugins.card.dl.sources;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import mage.MageException;
+import mage.client.util.CardLanguage;
+import mage.util.JsonUtil;
+import org.apache.log4j.Logger;
+import org.mage.plugins.card.dl.DownloadServiceInfo;
+import org.mage.plugins.card.images.CardDownloadData;
+import org.mage.plugins.card.dl.sources.ScryfallImageSource;
+
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.Proxy;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.*;
+
+/**
+ * @author tiera3
+ */
+public enum ScryfallImageSourceSmall implements CardImageSource {
+    instance;
+	
+    private CardImageSource baseSource = ScryfallImageSource.instance;
```

Comment:
@JayDi85 what do you think of this style? I think it could make more sense to have `ScryfallImageSource.LARGE` and `ScryfallImageSource.SMALL` rather than this wrapper class. But it's not a big deal and this way should work.

Reply:
That sounds like what [Susucre](https://github.com/Susucre) suggested in my original PR https://github.com/magefree/mage/pull/12263 .

My reasons for not doing so were:
a) I wasn't sure how to, and
b) I didn't want to change ScryfallImageSource.java because of other open PRs.

Reply:
I’m surprised why it’s not class with overrided methods or constructor param. You can use it here cause already merged in other PR, but I will refactor and remove that “base source” anyway.

Reply:
Hmm, it’s not in master yet.

Reply:
Make original enum as class and use create it here. It’s ok to combine enum and class objects:

![IMG_0426](https://github.com/magefree/mage/assets/8344157/32d3462e-ca50-4f96-9cc7-a76afe304c65)

Reply:
Or getInstance to simulate enum logic:

![IMG_0427](https://github.com/magefree/mage/assets/8344157/1e23c656-0621-4caa-8c73-c2be3ceda9cd)

Reply:
Example:
ScryfallImageSource - class with static “instance” field and “getInstance” method
ScryfallImageSourceSmall extends ScryfallImageSource with overrided modify and size methods